eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -60,7 +60,12 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <module name="LocalFinalVariableName"/>
     <module name="LocalVariableName"/>
     <module name="MemberName"/>
-    <module name="MethodName"/>
+    <module name="MethodName">
+      <!-- Use default MethodName but block the use of 'builder' as method name -->
+      <property name="format" value="(?=^[a-z][a-zA-Z0-9]*$)(?!^[b][uU][iI][lL][dD][eE][rR]$)"/>
```

Comment:
Are mode specifiers available? It's easier to read `((?i)builder)$

Reply:
Yes, it is supported. I have tested and worked.
Thank you for it! 👍  Updated now